go/samples/mcp-client/main.go

```diff
@@ -250,13 +251,74 @@ func clientGetPromptExample() {
 	}
 }
 
+// MCP Client Streamable HTTP Example - connects to a server via Streamable HTTP transport
+func clientStreamableHTTPExample() {
+	ctx := context.Background()
+
+	// Initialize Genkit with Google AI
+	g, err := genkit.Init(ctx, genkit.WithPlugins(&googlegenai.GoogleAI{}))
+	if err != nil {
+		logger.FromContext(ctx).Error("Failed to initialize Genkit", "error", err)
+		return
+	}
+
+	logger.FromContext(ctx).Info("Creating MCP client with Streamable HTTP transport", "server", "everything")
+	// Create and connect to MCP server using Streamable HTTP transport
+	// Note: Start the server with: npx @modelcontextprotocol/server-everything streamableHttp --port 3001
+	// This will start the server on http://localhost:3001
+	client, err := mcp.NewGenkitMCPClient(mcp.MCPClientOptions{
+		Name:    "mcp-everything-http",
+		Version: "1.0.0",
+		StreamableHTTP: &mcp.StreamableHTTPConfig{
+			BaseURL: "http://localhost:3001",
+			Headers: map[string]string{
+				"User-Agent": "genkit-mcp-client/1.0.0",
```

Comment:
Should this be the Genkit Go version? We have it available in `internal` package.